demo/prepare_xpack.py

```diff
@@ -0,0 +1,46 @@
+from elasticsearch_xpack import XPackClient
```

Comment:
should we also have this script as part of some bin directory, some utility to setup the plugin with xpack?